src/Aspire.Dashboard/Model/ResourceViewModel.cs

```diff
@@ -166,8 +168,35 @@ public sealed class CommandViewModel
 {
     public string Name { get; }
     public CommandViewModelState State { get; }
-    public string DisplayName { get; }
-    public string DisplayDescription { get; }
+
+    public string DisplayName
+    {
+        get
+        {
+            return Name switch
+            {
+                KnownResourceCommands.StartCommand => Commands.StartCommandDisplayName,
+                KnownResourceCommands.StopCommand => Commands.StopCommandDisplayName,
+                KnownResourceCommands.RestartCommand => Commands.RestartCommandDisplayName,
+                _ => field
+            };
+        }
+    }
+
+    public string DisplayDescription
+    {
+        get
+        {
+            return Name switch
+            {
+                KnownResourceCommands.StartCommand => Commands.StartCommandDisplayDescription,
+                KnownResourceCommands.StopCommand => Commands.StopCommandDisplayDescription,
+                KnownResourceCommands.RestartCommand => Commands.RestartCommandDisplayDescription,
+                _ => field
+            };
+        }
+    }
```

Comment:
We want to access resource translations via IStringLocalizer.

Can you change these to methods to methods that have an `IStringLocalizer<Commands>` parameter. Then whoever uses them is responsible for passing the localizer